cypress/integration/landing_page_spec.ts

```diff
@@ -7,24 +7,20 @@ describe('Landing Page test suite', () => {
 
   describe('Content', () => {
 
-    it('header displays ngx-bootstrap logo and info buttons', () => {
-      cy.get(landing.logoAtHeader)
-        .should('be.visible');
-      cy.get(landing.infoButtons)
-        .should('be.visible');
+    it('header displays ngx-bootstrap logo and info buttons to the: stackoverflow, gitHub and slack', () => {
+      landing.isElementVisible(landing.headerSelector, landing.logoAtHeader);
+      landing.isElementVisible(landing.headerSelector, landing.infoButtons);
+      landing.isElementVisible(landing.headerSelector, landing.stackOverBtn);
+      landing.isElementVisible(landing.headerSelector, landing.gitHbBtn);
+      landing.isElementVisible(landing.headerSelector, landing.slackNgxBtn);
     });
 
     it('main content displays ngx-bootstrap logo, slogan, description, version and advantages block', () => {
-      cy.get(landing.logoAtContent)
-        .should('be.visible');
-      cy.get(landing.sloganBs)
-        .should('be.visible');
-      cy.get(landing.descriptionBs)
-        .should('be.visible');
-      cy.get(landing.versionBs)
-        .should('be.visible');
-      cy.get(landing.advantagesBs)
-        .should('be.visible');
+      landing.isElementVisible(landing.mainClass, landing.logoAtContent);
```

Comment:
move this checks into previous 'it'. If you don't make any actions, and make only assertions - better to serve them in 1 'it'